src/UI/class-settings-page.php

```diff
@@ -285,41 +287,19 @@ function (): void {
 		);
 
 		// Allow use of custom taxonomy to populate articleSection in parselyPage; defaults to category.
-		$h          = __( 'By default, Parse.ly only tracks the default post type as a post page. If you want to track custom post types, select them here!', 'wp-parsely' );
-		$field_id   = 'track_post_types';
-		$field_args = array(
-			'title'          => __( 'Post Types to Track', 'wp-parsely' ),
-			'option_key'     => $field_id,
-			'help_text'      => $h,
-			'select_options' => get_post_types( array( 'public' => true ) ),
-			'label_for'      => Parsely::OPTIONS_KEY . "[$field_id]",
-		);
+		$field_id   = 'track_post_types_as';
+		$field_help = __( 'Select how you want to track every Post Type.', 'wp-parsely' );
 		add_settings_field(
 			$field_id,
-			__( 'Post Types to Track', 'wp-parsely' ),
-			array( $this, 'print_multiple_checkboxes' ),
-			Parsely::MENU_SLUG,
-			'requires_recrawl_settings',
-			$field_args
-		);
-
-		// Allow use of custom taxonomy to populate articleSection in parselyPage; defaults to category.
-		$h          = __( 'By default, Parse.ly only tracks the default page type as a non-post page. If you want to track custom post types as non-post pages, select them here!', 'wp-parsely' );
-		$field_id   = 'track_page_types';
-		$field_args = array(
-			'title'          => __( 'Page Types to Track', 'wp-parsely' ),
-			'option_key'     => 'track_page_types',
-			'help_text'      => $h,
-			'select_options' => get_post_types( array( 'public' => true ) ),
-			'label_for'      => Parsely::OPTIONS_KEY . "[$field_id]",
-		);
-		add_settings_field(
-			'track_page_types',
-			__( 'Page Types to Track', 'wp-parsely' ),
-			array( $this, 'print_multiple_checkboxes' ),
+			__( 'Track Post Types as...', 'wp-parsely' ),
```

Comment:
I would remove the `...` from the label, so leave it as `Track Post Types as`

Reply:
Not pushed yet, but done.